back/api/routers/crud.py

```
@@ -1,102 +1,102 @@
+    updated_book = session.query(book_model.Books).filter(book_model.Books.id == book_id).first()
+    if updated_book is None:
+        raise HTTPException(status_code=404, detail="Book not found")
+    updated_book.cost = cost_update.cost
+    updated_book.last_modified = datetime.now(timezone.utc)
+    session.commit()
+    session.refresh(updated_book)
+    return updated_book
+
+# ステータスを更新 
+@router.patch("/{book_id}/status", response_model=books.Book) 
+def update_book_detail_status(book_id: int, status_update: books.BookStatusUpdate, session: Session = Depends(connection.get_db)):
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)